src/buildblock/ProjDataInfo.cxx

```diff
@@ -432,16 +432,15 @@ ProjDataInfo::get_empty_segment_by_view(const int segment_num,
 }
 
 RelatedViewgrams<float> 
-ProjDataInfo::get_empty_related_viewgrams(const ViewSegmentNumbers& view_segment_num,
+ProjDataInfo::get_empty_related_viewgrams(const ViewSegmentTOFNumbers& view_segment_num,
                    //const int view_num, const int segment_num,
 		   const shared_ptr<DataSymmetriesForViewSegmentNumbers>& symmetries_used,
-		   const bool make_num_tangential_poss_odd,
-		   const int timing_pos_num) const
+		   const bool make_num_tangential_poss_odd) const
 {
-  vector<ViewSegmentNumbers> pairs;
+  vector<ViewSegmentTOFNumbers> pairs;
   symmetries_used->get_related_view_segment_numbers(
                                                     pairs, 
-                                                    ViewSegmentNumbers(view_segment_num.view_num(),view_segment_num.segment_num())
+                                                    ViewSegmentTOFNumbers(view_segment_num.view_num(),view_segment_num.segment_num())
     );
```

Comment:
Why construct a new `ViewSegmentTOFNumbers` here? pass `view_segment_num`? This comes from [UCL/STIR's projdatainfo.cxx ](https://github.com/UCL/STIR/blob/6312c07f1fdd50eb48eacdec12801f66525221e4/src/buildblock/ProjDataInfo.cxx#L311-L314)

Reply:
Sorry, I was only suppose to highlight  

```
symmetries_used->get_related_view_segment_numbers(
                                                    pairs, 
                                                    ViewSegmentNumbers(view_segment_num.view_num(),view_segment_num.segment_num())
                                                    ViewSegmentTOFNumbers(view_segment_num.view_num(),view_segment_num.segment_num())
    );
```

Reply:
sure. no need to make a new one, just pass it along